app/models/rating.js

```diff
@@ -0,0 +1,31 @@
+module.exports = (sequelize, DataTypes) => {
+  const Rating = sequelize.define(
+    'Rating',
+    {
+      ratingUserId: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        onUpdate: 'cascade',
+        onDelete: 'cascade'
```

Comment:
It's not part of the card, but if someone vote me up, if that user deletes his profile I don't want it to affect my rating